### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by CTJohnson** [timeoff-management/timeoff-management-application#501]
Is the project dead?

**Comment by squeakus** [comment]
I don't know if it is dead, but it is certainly quiet...

**Comment by DooMMasteR** [comment]
yeah not even the API stuff saw any work so far, and it would be essential to do at least some automation....

**Comment by Kurren123** [comment]
Yes it looks unmaintained. Lots of issues with no reply, 40+ unmerged PRs, last commit was 2 years ago.

**Comment by bigheadsmith** [comment]
There is a fork by aliengen then a fork of that by ashlessscythe which has seen more recent development.

I think the original project developer has focused on their paid model
